runtime/error.c

```diff
@@ -17,6 +17,13 @@
 #define EXCEEDS_MAX_PC_COUNT -3
 #define OUT_OF_MEMORY -4
 
+#define PUBLIC_BALLERINA_NAME 1
```

Comment:
Use a sensible set of numbers. Those were just the numbers in my list of possible cases.

Reply:
Changed.